tasks/9_find-numbers/find-numbers.js

```diff
@@ -0,0 +1,5 @@
+function getNumbers(str) {
+  return str.match(/\d/g).map(Number);
```

Comment:
it's better to avoid passing function directly.
```map(number => Number(number));```
why? take a look at output of that code:
```javascript
['2','2','2','2'].map(parseInt);
```